stan/math/rev/functor/cvodes_utils.hpp

```diff
@@ -10,38 +10,34 @@
 namespace stan {
 namespace math {
 
-extern "C" inline void cvodes_err_handler(int error_code, const char* module,
-                                          const char* function, char* msg,
-                                          void* eh_data) {
-  if (error_code != CV_TOO_MUCH_WORK) {
-    std::ostringstream msg1;
-    msg1 << msg << " Error code: ";
-
-    throw_domain_error(module, function, error_code, msg1.str().c_str());
-  }
-}
+// extern "C" inline void cvodes_err_handler(int error_code, const char* module,
+//                                           const char* function, char* msg,
+//                                           void* eh_data) {
+//   if (error_code != CV_TOO_MUCH_WORK) {
+//     std::ostringstream msg1;
+//     msg1 << msg << " Error code: ";
+
+//     throw_domain_error(module, function, error_code, msg1.str().c_str());
+//   }
+// }
```

Comment:
If this code is not needed, lets just remove it.

Reply:
done, thanks.